weeklyChallenges02/ChallengesSet02.cs

```diff
@@ -6,56 +6,91 @@ namespace weeklyChallenges02
 {
     public class ChallengesSet02
     {
-        public bool CharacterIsALetter(char c)
-        {
-            throw new NotImplementedException();
-        }
+        public bool CharacterIsALetter(char c) => Char.IsLetter(c);
 
-        public bool CountOfElementsIsEven(string[] vals)
-        {
-            throw new NotImplementedException();
-        }
+        public bool CountOfElementsIsEven(string[] vals) => vals.Length % 2 == 0;
 
-        public bool IsNumberEven(int number)
-        {
-            throw new NotImplementedException();
-        }
+        public bool IsNumberEven(int number) => number % 2 == 0;
 
 
-        public bool IsNumberOdd(int num)
-        {
-            throw new NotImplementedException();
-        }
+        public bool IsNumberOdd(int num) => num % 2 != 0;
 
         public double SumOfMinAndMax(IEnumerable<double> numbers)
         {
-            throw new NotImplementedException();
+            if (numbers == null)
```

Comment:
Consider combining the if & else if statement together on one line, using an or `||` to shorten the code. other than that, good work